Engine/Setup/BacktestingSetupHandler.cs

```diff
@@ -262,17 +245,51 @@ public bool Setup(SetupHandlerParameters parameters)
             return initializeComplete;
         }
 
+        /// <summary>
+        /// Get maximum orders for our algorithm, restricted by UserPlan
+        /// </summary>
+        /// <param name="job"></param>
+        /// <returns></returns>
+        protected virtual int GetMaximumOrders(BacktestNodePacket job)
+        {
+            if (job == null)
+            {
+                throw new ArgumentException("Job must not be null");
+            }
+
+            //Max Orders: 10k per backtest:
+            if (job?.UserPlan == UserPlan.Free)
+            {
+                MaxOrders = 10000;
+            }
+            else
+            {
+                MaxOrders = int.MaxValue;
+                MaximumRuntime += MaximumRuntime;
+            }
+
+            MaxOrders = job.Controls.BacktestingMaxOrders;
```

Comment:
This looks fishy, `MaxOrders` value is overriden by `job.Controls.BacktestingMaxOrders` so what's the `if/else` above for? `MaximumRuntime += MaximumRuntime;` is also fishy 😁 should be in the call to `GetMaximumRuntime()` I think

Reply:
💯